boards/DCM/Inc/App/App_Imu.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+#include <stdint.h>
+#include <stdbool.h>
+#include <App_SharedExitCode.h>
+
+struct Imu;
+
+/**
+ * Creates Imu struct used for reading data from sensor
+ * @param update_sensor_data function which updates data and returns
+ * EXIT_CODE_OK if successful, EXIT_CODE_OUT_OF_RANGE if data is not within
+ * specified range, EXIT_CODE_INVALID_ARGS if data not read successfully
+ * @param get_acceleration_x A function that can be called to get the
+ * acceleration (m/s^2) on the x-axis.
+ * @param get_acceleration_y A function that can be called to get the
+ * acceleration (m/s^2) on the y-axis.
+ * @param get_acceleration_z A function that can be called to get the
+ * acceleration (m/s^2) on the z-axis.
+ * @return Imu struct with functions above
+ */
+struct Imu *App_Imu_Create(
+    ExitCode (*update_sensor_data)(void),
+    float (*get_acceleration_x)(void),
+    float (*get_acceleration_y)(void),
+    float (*get_acceleration_z)(void));
+
```

Comment:
You should also add a App_Imu_Destroy function

Reply:
done